### DISCUSSION TITLE
Precache

### DISCUSSION THREAD

**Discussion by og900aero** [sxyazi/yazi#730] [1. Q&A] [answered]
Is it possible to do the caching with Yazi in advance? Let's say that I would give it the path to recursively create the cache in advance (coloring images, files/folders), so that I don't always have to wait for it to figure out what color the files and folders should be displayed with, and for the preview of the image to appear.

**Comment by sxyazi** [comment, marked as answer]
Check out https://github.com/DreamMaoMao/mime.yazi

**Reply by og900aero** [reply]
Whoaaa, working perfect! Thx. I also added the audio dsf format to the plugin.
But my opinion, is must be default "settings" in yazi code, not plugin.

**Reply by sxyazi** [reply]
I want to avoid making breaking changes, files like [`.prettierrc`](https://prettier.io/docs/en/configuration.html) can be recognized as JSON currently through `file(1)` but this plugin doesn't, this will impact some users who rely on this behavior.

**Reply by og900aero** [reply]
OK, no problem. I just thought it was a very good plugin, because it's quite annoying that it loads the colors slowly anyway. That's why I thought it would be great if you knew this from the ground up.
